python/pyspark/sql/__init__.py

```diff
@@ -51,12 +51,12 @@
 from pyspark.sql.group import GroupedData
 from pyspark.sql.readwriter import DataFrameReader, DataFrameWriter
 from pyspark.sql.window import Window, WindowSpec
-from pyspark.sql.cogroup import CoGroupedData
+from pyspark.sql.pandas.group_ops import PandasCogroupedOps
```

Comment:
Just a question. It's `PandasCogroupedOps` instead of `PandasCoGroupedOps` intentionally, right?

Reply:
Ah yes, I brought such nit fixes together (where I am almost 100% sure) given the naming in our official doc (https://github.com/apache/spark/blob/f31d9a629b94f5db57bdaf96eae90f165c588da6/docs/sql-pyspark-pandas-with-arrow.md#cogrouped-map)